cliquet/cache/__init__.py

```diff
@@ -85,6 +86,7 @@ def ping(request):
                 backend.set(_HEARTBEAT_KEY, 'alive', _HEARTBEAT_TTL_SECONDS)
             return True
         except:
+            logger.exception("Heartbeat Failure")
```

Comment:
logger.exception is dumping the full exception info. see https://docs.python.org/2/library/logging.html#logging.Logger.exception

What would you like to see changed ?